ExampleMod/Common/Players/ExampleRegenPlayer.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Terraria;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Common.Players
+{
+	public class ExampleRegenPlayer : ModPlayer
+	{
+		public override ModPlayer.Regen ModifyLifeRegen() {
+			var modifiers = ModPlayer.Regen.Create(); // Creates the struct with all values configured to do no modifications.
+			modifiers.deltaRate = 3f; // Increases the rate at which player reaches maximum natural life regeneration for every multiple of 300 up to 3600. Positives for buffs, negatives for debuffs.
```

Comment:
comments about how the field works above the line. Comments about exactly what this line does, on the line.

I can't actually tell what this line does... it's not a multiple of 300 so.. nothing?

Reply:
Fixed. That line did nothing - that's why I had asked in Discord for anyone with an opinion on what the rate should be doing relative to their expectations.
Now its setup meaningfully, I think.